editor.planx.uk/src/theme.ts

```diff
@@ -37,6 +37,9 @@ const DEFAULT_PALETTE: Partial<PaletteOptions> = {
   error: {
     main: "#E91B0C",
   },
+  success: {
+    main: "#00703C",
+  },
```

Comment:
nit: Is this used currently?

Reply:
`theme.palette.success.main` is not referenced anywhere explicitly in our code, and i did click around a bit after setting this to ensure it wasn't being referenced under the hood by MUI for default component states and didn't spot any - so I think this is safe to introduce! it felt a bit cleaner and easier to reference in future than hardcoding the "completed" green shade exclusively in the Sections component.